tests/dCommonTests/dEnumsTests/MagicEnumTests.cpp

```diff
@@ -140,3 +140,24 @@ TEST(MagicEnumTest, ArraysAreSorted) {
 
 	delete Game::logger;
 }
+
+#define IS_CONSTEXPR2(...) std::integral_constant<bool, __builtin_constant_p(([] () { __VA_ARGS__; } (), 0))>::value
```

Comment:
Im not sure if im a fan of using an internal compiler function for this.  This is non-standard and will cause issues on other platforms that do not implement this function

Reply:
Ah. That wasn't even supposed to be in there.